connector_jira/components/jira_analytic_line_mapper.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2016-2022 Camptocamp SA
+# Copyright 2019 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from pytz import timezone, utc
+
+from odoo import _
+
+from odoo.addons.component.core import Component
+from odoo.addons.connector.components.mapper import mapping
+from odoo.addons.connector.exception import MappingError
+
+from .common import iso8601_to_naive_date, iso8601_to_utc_datetime, whenempty
+
+
+class JiraAnalyticLineMapper(Component):
+    _name = "jira.analytic.line.mapper"
+    _inherit = "jira.import.mapper"
+    _apply_on = ["jira.account.analytic.line"]
+
+    direct = [(whenempty("comment", _("missing description")), "name")]
```

Comment:
I'm not sure, if this should prevent issue for receiving Time logs from Jira without description, but there is still an error in such cases:
```
Exception:?KeyError
Traceback (most recent call last):
  File "/home/odoo/src/user/oca-queue/queue_job/controllers/main.py", line 104, in runjob
    self._try_perform_job(env, job)
  File "/home/odoo/src/user/oca-queue/queue_job/controllers/main.py", line 36, in _try_perform_job
    job.perform()
  File "/home/odoo/src/user/oca-queue/queue_job/job.py", line 517, in perform
    self.result = self.func(*tuple(self.args), **self.kwargs)
  File "/home/odoo/src/user/connector_jira/models/jira_account_analytic_line.py", line 88, in import_record
    return importer.run(worklog_id, issue_id=issue_id, force=force)
  File "/home/odoo/src/user/connector_jira/components/jira_analytic_line_importer.py", line 119, in run
    return super().run(external_id, force=force, record=record, **kwargs)
  File "/home/odoo/src/user/connector_jira/components/jira_importer.py", line 364, in run
    self._import(binding, **kwargs)
  File "/home/odoo/src/user/connector_jira/components/jira_analytic_line_importer.py", line 164, in _import
    return super()._import(binding, **kwargs)
  File "/home/odoo/src/user/connector_jira/components/jira_importer.py", line 379, in _import
    record = self._create_data(map_record)
  File "/home/odoo/src/user/connector_jira/components/jira_analytic_line_importer.py", line 93, in _create_data
    return super()._create_data(
  File "/home/odoo/src/user/connector_jira/components/jira_importer.py", line 155, in _create_data
    return map_record.values(
  File "/home/odoo/src/user/oca-connector/connector/components/mapper.py", line 1016, in values
    values = self._mapper._apply(self, options=options)
  File "/home/odoo/src/user/oca-connector/connector/components/mapper.py", line 786, in _apply
    return self._apply_with_options(map_record)
  File "/home/odoo/src/user/oca-connector/connector/components/mapper.py", line 813, in _apply_with_options
    value = self._map_direct(map_record.source, from_attr, to_attr)
  File "/home/odoo/src/user/oca-connector/connector/components/mapper.py", line 882, in _map_direct
    return from_attr(self, record, to_attr)
  File "/home/odoo/src/user/connector_jira/components/common.py", line 156, in modifier
    value = record[field]
KeyError: 'comment'
```


in function whenempty() first arg is:
:param field: name of the source field in the record

which means, should be "name" not a comment, right?